src/main/java/com/naitei/group3/movie_ticket_booking_system/controller/admin/MovieController.java

```diff
@@ -128,6 +131,20 @@ private String validateFile(MultipartFile file) {
         return null;
     }
 
+    @GetMapping("/{id}/update")
```

Comment:
```suggestion
    @GetMapping("/{id}/edit")
```
theo chuẩn thi ri mới đúng nha